jbmc/src/java_bytecode/java_object_factory.cpp

```diff
@@ -1591,3 +1626,29 @@ void gen_nondet_init(
     update_in_place,
     log);
 }
+std::pair<code_blockt, symbol_exprt> nondet_array(
```

Comment:
Unused here, so needs a test

Reply:
Blast you're right - might move it into TG for now.

Reply:
This ends up going away - the whole free function is now used by the Java Object Factory.